libs/dh/admin/feature-user-management/src/lib/invite-user-modal/dh-invite-user-modal.component.html

```diff
@@ -14,4 +14,74 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<watt-modal #inviteUserModal> invite user </watt-modal>
+﻿<ng-container *transloco="let t; read: 'admin.userManagement.inviteUser'">
```

Comment:
Suggestion: Could `*transloco` just be added to `watt-modal` to avoid the `ng-container`?